test/integration/targets/ecs_cluster/runme.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
+# We don't set -u here, due to pypa/virtualenv#150
+set -ex
+
+MYTMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
+
+trap 'rm -rf "${MYTMPDIR}"' EXIT
+
+# This is needed for the ubuntu1604py3 tests
+# Ubuntu patches virtualenv to make the default python2
+# but for the python3 tests we need virtualenv to use python3
+PYTHON=$("python${ANSIBLE_TEST_PYTHON_VERSION:-}" -c "import sys; print(sys.executable)")
```

Comment:
You should be able to use `$ANSIBLE_TEST_PYTHON_INTERPRETER` instead. I don't recall why our existing tests use `$ANSIBLE_TEST_PYTHON_VERSION` to find the interpreter.